cmake/modules/platform/toolcfg/openxl.cmake

```diff
@@ -0,0 +1,180 @@
+###############################################################################
+# Copyright IBM Corp. and others 2024
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+###############################################################################
+
+if(CMAKE_C_COMPILER_IS_XLCLANG)
+	macro(omr_toolconfig_global_setup)
+		# For XLClang, remove any usages of -qhalt=e or -qhalt=s provided by default
+		# in the CMAKE CXX/C/ASM FLAGS, since xlclang/xlclang++ are not compatible
+		# with the e or s options.
+		omr_remove_flags(CMAKE_ASM_FLAGS -qhalt=e)
+		omr_remove_flags(CMAKE_C_FLAGS   -qhalt=e)
+		omr_remove_flags(CMAKE_CXX_FLAGS -qhalt=s)
+	endmacro(omr_toolconfig_global_setup)
+endif()
```

Comment:
This file should not be used when `CMAKE_C_COMPILER_IS_XLCLANG` (in which case `OMR_TOOLCONFIG` is `xlc`). All the resulting dead code should be removed from this file.

Even after such changes this would still use `-q` options, which I believe are for xlc, not OpenXL.

Reply:
Thanks, removed these from the toolcfg file.